cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -0,0 +1,1112 @@
+const $ = require( 'jquery' );
+const amortize = require( 'amortize' );
+const formatUSD = require( 'format-usd' );
+const isNum = require( 'is-money-usd' );
+const jumbo = require( 'jumbo-mortgage' );
+const median = require( 'median' );
+const unFormatUSD = require( 'unformat-usd' );
+
+// Load and style Highcharts library. https://www.highcharts.com/docs.
+const Highcharts = require( 'highcharts' );
+require( 'highcharts/modules/exporting' )( Highcharts );
+const highchartsTheme = require( './highcharts-theme' );
+highchartsTheme.applyThemeTo( Highcharts );
+
+// var geolocation = require('./geolocation');
+const config = require( '../../config.json' );
+const domValues = require( './dom-values' );
+const dropdown = require( '../dropdown-utils' );
+const fetchRates = require( '../rates' );
+const formatTime = require( '../format-timestamp' );
+const params = require( './params' );
+
+require( 'rangeslider.js' );
+require( './tab' );
+require( '../placeholder-polyfill' );
+
+// Load our handlebar templates.
+const template = require( './template-loader' );
+
+import { uniquePrimitives } from '../../../../js/modules/util/array-helpers';
```

Comment:
How would you feel about aliasing the utils directory? It would be `UTIL/array-helpers`.

Reply:
That would be done in webpack?